gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/DateTime.java

```diff
@@ -34,7 +32,7 @@
  * @see <a href="https://cloud.google.com/datastore/docs/concepts/entities">Google Cloud Datastore
  *     Entities, Properties, and Keys</a>
  */
-public final class DateTime extends Serializable<DatastoreV1.Value>
+public final class DateTime extends Serializable<com.google.datastore.v1beta3.Value>
```

Comment:
I think we should change DateTime to extends Serializable<com.google.protobuf.Timestamp>